packages/sdks/snippets/solidjs/src/components/AnnouncementBar.tsx

```diff
@@ -30,11 +30,11 @@ function AnnouncementBar() {
 
   return (
     <>
-      {content() ? (
+      {/* Your Announcement Bar goes here */}
```

Comment:
I don't think all of the "your announcement bar goes here" comments are needed. the code below _is_ the announcement bar code, I would just delete all of these comments

Reply:
Resolving this as we discussed in the slack.